tests/e2e/data/observatory/enrichment/test_enrichment.py

```diff
@@ -13,9 +14,26 @@ def file_path(path):
 
 def clean_gdf(gdf, sort_column=None):
     if sort_column:
-        return gdf.sort_index(axis=1).sort_values(by=sort_column).round(3).reset_index(drop=True)
+        return gdf.sort_index(axis=1).sort_values(by=sort_column).reset_index(drop=True)
     else:
-        return gdf.sort_index(axis=1).round(3).reset_index(drop=True)
+        return gdf.sort_index(axis=1).reset_index(drop=True)
+
+
+def assert_df_equals(enriched_gdf, expected_gdf):
+    assert (enriched_gdf.columns == expected_gdf.columns).all()
+    assert (enriched_gdf.count() == expected_gdf.count()).all()
+
+    for i in expected_gdf:
+        enriched_values = enriched_gdf[i].values
+        expected_values = expected_gdf[i].values
+        if isinstance(expected_values[0], str):
+            assert enriched_values.equals(expected_values)
+        elif i == 'geometry':
+            assert enriched_values.equals(expected_values).all()
+        elif expected_values.all() is None:
+            assert enriched_values.all() is None
+        else:
+            assert np.isclose(np.nan_to_num(enriched_values), np.nan_to_num(expected_values), rtol=0.01).all()
```

Comment:
We can add this `rtol` as a parameter with a default value. We can try to lower its value to `0.001`, if possible.